File: OnlineMedicineBookingApplication.Application/Models/CartDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OnlineMedicineBookingApplication.Application.Models
{
    public class CartDTO
    {
        public int CartId { get; set; }
        public int UserId { get; set; }
        public required List<CartItemDTO> Items { get; set; }
        public decimal TotalPrice { get; set; }
    }
}

File: MediPick/src/app/DTOS/cart-dto.ts
export class CartDto {
    cartId: number=0;
    userId: number=0;
    items: CartItemDTO[]=[];
    totalPrice: number=0;
}
export class CartItemDTO {
    mmedicneName : string = '';
    medicineId : number = 0;
    quantity : number = 0;
}
